resources/zib/zib-BodyHeight.xml

```diff
@@ -0,0 +1,133 @@
+  <id value="zib-BodyHeight" />
+  <url value="http://nictiz.nl/fhir/StructureDefinition/zib-BodyHeight" />
+  <name value="ZibBodyHeight" />
+  <title value="zib BodyHeight" />
+  <status value="draft" />
+  <publisher value="Nictiz" />
+  <contact>
+    <name value="Nictiz" />
+    <telecom>
+      <system value="email" />
+      <value value="info@nictiz.nl" />
+      <use value="work" />
+    </telecom>
+  </contact>
+  <description value="A person’s body height." />
+  <purpose value="This Observation resource represents the Dutch [zib ('Zorginformatiebouwsteen', i.e. Health and Care Information Model) BodyHeight v3.1.1(2020)](https://zibs.nl/wiki/BodyHeight-v3.1.1(2020EN))." />
+  <copyright value="Copyright and related rights waived via CC0, https://creativecommons.org/publicdomain/zero/1.0/. This does not apply to information from third parties, for example a medical terminology system. The implementer alone is responsible for identifying and obtaining any necessary licenses or authorizations to utilize third party IP in connection with the specification or otherwise." />
+  <fhirVersion value="4.0.1" />
+  <mapping>
+    <identity value="zib-bodyheight-v3.1.1-2020EN" />
+    <uri value="https://zibs.nl/wiki/BodyHeight-v3.1.1(2020EN)" />
+    <name value="zib BodyHeight-v3.1.1(2020EN)" />
+  </mapping>
+  <kind value="resource" />
+  <abstract value="true" />
+  <type value="Observation" />
+  <baseDefinition value="http://hl7.org/fhir/StructureDefinition/Observation" />
+  <derivation value="constraint" />
+  <differential>
+    <element id="Observation">
+      <path value="Observation" />
+      <short value="BodyHeight" />
+      <alias value="Lichaamslengte" />
+      <mapping>
+        <identity value="zib-bodyheight-v3.1.1-2020EN" />
+        <map value="NL-CM:12.2.1" />
+        <comment value="BodyHeight" />
+      </mapping>
+    </element>
+    <element id="Observation.effective[x]">
+      <path value="Observation.effective[x]" />
+      <short value="HeightDateTime" />
+      <definition value="Date and if possible time of body height measurement." />
+      <alias value="LengteDatumTijd" />
+      <min value="1" />
+      <type>
+        <code value="dateTime" />
+      </type>
+      <type>
+        <code value="Period" />
+      </type>
+      <mapping>
+        <identity value="zib-bodyheight-v3.1.1-2020EN" />
+        <map value="NL-CM:12.2.4" />
+        <comment value="HeightDateTime" />
+      </mapping>
+    </element>
+    <element id="Observation.value[x]">
+      <path value="Observation.value[x]" />
+      <short value="HeightValue" />
+      <definition value="The element contains the value of the measured body height.  &#xD;&#xA;Body height is the height of the human body from heel to scalp, generally measured when standing. Body height is measured in a stretched, horizontal position for very small children or patients that are unable to stand." />
+      <alias value="LengteWaarde" />
+      <min value="1" />
```

Comment:
niet conform zib conceptuele kardinaliteiten

Reply:
Eventueel met team overleggen over kardinaliteit voor .value[x] in nl-core laag gezet dient te worden want die lijkt wel heel logisch.

Reply:
Overigens is dat ook niet het geval in de Vital Signs profilen. Daar staat dat je ofwel de .valueQuanity moet doen of een .dataAbsentReason. Dat lijkt me wel legit voor in de nl-core- laag eigenlijk.

Reply:
Changed to 0 .. 1 in the zib